QuackNBOOM/QuackNBOOM/GameScene.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  GameScene.swift
+//  QuackNBOOM
+//
+//  Created by Ngo Tuyetnhi N. and Benoit Neriah on 3/4/18.
+//  Copyright © 2018 Neriah and Neenee. All rights reserved.
+//
+
+import SpriteKit
+import GameplayKit
+
+class GameScene: SKScene {
+    var deltaTime: TimeInterval = 0.0
+    private var lastUpdateTime: TimeInterval?
+    
+    ///just setting up the scene
+    
+    //creating variables
+    let background = GameObject(imagePath: "Background")
+    //let duckSprite = Duck(imagePath: "Duck")
+    //let bombSprite = Bomb(imagePath: "Bomb")
+    let breadSprite = Bread(imagePath: "Bread")
+    let gameOver = GameObject(imagePath: "GameOver")
+    let bombController = BombController()
+    let duckController = DuckController()
+    
+    override func didMove(to view: SKView) {
+        //adding the background to the scene
+        addChild(background.sprite)
+        background.pos = CGPoint(x: size.width/2, y: size.height/2)
+        
+        //adding the duck to the scene
+        //addChild(duckSprite.sprite)
+        //duckSprite.pos = CGPoint(x: size.width/2, y: 1000)
+        addChild(duckController.getAllDucks().sprite)
+        
+        //adding the bomb to the scene
+        //addChild(bombSprite.sprite)
+        //addChild(bombSprite.explosionSprite)
+        //bombSprite.pos = CGPoint(x: 750, y: 150)
+        addChild(bombController.getAllBombs().sprite)
+        addChild(bombController.getAllBombs().explosionSprite)
+        
+        //adding the bread to the scene
+        addChild(breadSprite.sprite)
+        breadSprite.pos = CGPoint(x: size.width/2, y: 100)
+        
+        //addChild(gameOver.sprite)
+        //gameOver.pos = CGPoint(x: size.width/2, y: size.height/2)
+        //gameOver.zPos = 3.0
+    }
+    
+    override func update(_ currentTime: TimeInterval) {
+        guard let lastUpdateTime = lastUpdateTime else {
+            self.lastUpdateTime = currentTime
+            return
+        }
+        
+        // Calculate deltaTime
+        deltaTime = currentTime - lastUpdateTime
```

Comment:
deltaTime is only used within this function, doesn't need to be a member variable at the class level, it could just be computer here.

Reply:
***This requested change has been successfully implemented***
The delta time has been moved into the update function now